src/components/CursorPaginationManager/CursorPaginationManager.tsx

```
@@ -0,0 +1,328 @@
+
+  // Error state (no items yet)
+  if (error && !items.length) {
+    return (
+      <div
+        role="alert"
+        aria-live="assertive"
+        data-testid="cursor-pagination-error"
+        className={styles.stateMessage}
+      >
+        <p>{error.message}</p>
+        <button onClick={handleRefetch} className={styles.loadMoreButton}>
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)